nsis/src/nsis/tests/NsisExecutableTest.java

```diff
@@ -62,9 +66,13 @@ public void testNsisCreationLZMACompressed() throws IOException, InvalidFormatEx
 			assertEquals(0x80, ne.getCommonHeaderFlags());
 			assertEquals(0x12c, ne.getBlockHeader(0).getOffset());
 			assertEquals(2, ne.getBlockHeader(0).getNumEntries());
+
+			// Pages
+			assertEquals(2, ne.getNumPages());
```

Comment:
Should we add one more check to include some data from within the page datastructure?

Also should we check one data element from each page? Like the `dialogResourceId` (or one of the other elements that is likely to change)?

If I understood our discussion about `getPageDataType()` above we added the page at offset 0 twice but the test did not catch that.

Reply:
Added tests for dialogResourceId for each page